reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+> We clearly say multiple times that these tradeoffs can not be optimized separately.
+> In the revised paper:
+> *As noted before, we strive for high detectability and high theoretical risk in planning, conducting, and analyzing studies.
+Maximizing one at the expense of the other is not necessarily beneficial for increasing epistemic value but depends on the specific function they apply to judge evidence and their specific location on the curve.
+One advantage of our framework is that researchers can employ it to balance the trade-off more effectively, assuming they are willing to make some simplifying assumptions.*
+
+POINT 4: Terminology As terminology can always be fixed, this point is not as problematic to the reasoning as the previous 3.
+The authors use the word 'theoretical risk' for the probability that a test of a hypothesis yields a true negative.
+As the authors exclusively talk about the probability that statistical tests will yield a correct conclusion, 'theoretical risk' feels as a misnomer (statistical risk would be better, but severity might also be a good general term).
+
+> We talk about the scientific procedures, of which a statistical test is a piece.
+
+True, in page 13 they give an example about theoretical risk that is actually theoretical risk - but this example has nothing to do with the remainder of the paper, as all three studies can have exactly the same alpha level!
+
+> We are unable to follow this argument without more elaboration.
+
+They are talking about a low frequentist error rate in all their examples, and as 'theoretical risk' is the inverse of the false positive rate, it might be easier to just talk about the Type 1 error rate.
+
+> The point of not using a Type 1 error rate is to differentiate the statistical test from the the researcher's scientific judgement.
+> Type 1 error rate is a property of a test under certain conditions.
+> Theoretical risk refers to the individual judgement of a researcher about the substantive hypothesis and the study design as a whole.
+
+The authors say they borrowed the term 'theoretical risk' from Meehl, but that he did not assign it to (¬\|¬).
+The authors cite Kukla (1990), but after reading it, I am not convinced they are correct in equating 'theoretical risk' with the Type 1 error rate.
+Theoretical risk means making an a-priori unlikely prediction if the theory was not true.
+
+> This sentence sounds exactly like P(¬E|¬H) to us.
+> The judgement about P(¬E|¬H) is a-priori, is about "prediction" as a statement about evidence E, given that the theory is not true (¬H).
+> One might argue, that it should be multiplied by ¬H perhaps? 
+> But that would not change any of the arguments.
+> In any case, we provide clear mathematical definitions for our terms.
+> If a reader does not agree with our terminology, they are free to term them differently.
```

Comment:
I don't like the tone of that argument here. Terms should be well grounded (but I think they are).